bigflow/commons.py

```diff
@@ -4,11 +4,11 @@
 import logging
 
 from pathlib import Path
-from datetime import datetime
-
 
 logger = logging.getLogger(__name__)
 
+DEFAULT_JOB_EXECUTION_TIMEOUT = 3600000  # 1 hour
```

Comment:
It used to be longer, keep the same execution timeout as it was to avoid breaking users' workflows.

Reply:
Ok

Reply:
Set to None as default

Reply:
Why None? I think it should be the same value as it used to be in the previous version.

Reply:
And if you set None as default then what's the final default timeout (it can't be just None)?

Reply:
> Why None? I think it should be the same value as it used to be in the previous version.
And this is value from previous version. Default values in dataflow and kubernetesPodOperator are None.

Reply:
True, previously the default timeout was inherited from the DAG settings.